pyro/infer/mcmc/api.py

```diff
@@ -385,13 +385,13 @@ def run(self, *args, **kwargs):
         # If transforms is not explicitly provided, infer automatically using
         # model args, kwargs.
         if self.transforms is None:
-            if hasattr(self.kernel, 'transforms'):
-                if self.kernel.transforms is not None:
-                    self.transforms = self.kernel.transforms
-            elif self.kernel.model:
-                _, _, self.transforms, _ = initialize_model(self.kernel.model,
-                                                            model_args=args,
-                                                            model_kwargs=kwargs)
+            if hasattr(self.kernel, 'transforms'):  # If the kernel has transforms
+                if self.kernel.transforms is not None:  # and they are not None
+                    self.transforms = self.kernel.transforms # then use the kernels transforms
```

Comment:
I think the lint is failing here. Btw, you could run it locally using `make lint`.